### PR TITLE
[CIS-821] Audit ChatMessageLinkPreviewView

### PR DESCRIPTION
**Pull request by fortmarek**
## Description of the pull request

- Audit ChatMessageLinkPreviewView
- Link preview injector

Note the screenshot tests look a little bit weird because `authorBackground` has all corners maskes - whereas it should be just top right one. This works well when you run the app but has issues in the snapshot testing as per [this](https://github.com/pointfreeco/swift-snapshot-testing/issues/358). I am not sure if I should go down the rabbit hole to try to fix this - or do not show the `authorBackground` for snapshot tests ... cc @VojtaStavik

### CODE REVIEW

File: Sources/StreamChatUI/Components.swift [hunk 1]
```diff
@@ -70,12 +70,15 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     /// The view used to display content of the message, i.e. in the channel detail message list.
     public var messageContentView: _ChatMessageContentView<ExtraData>.Type = _ChatMessageContentView<ExtraData>.self
 
-    /// The injector used to inject gallery attachment views
+    /// The injector used to inject gallery attachment views.
     public var galleryAttachmentInjector: _AttachmentViewInjector<ExtraData>.Type = _GalleryAttachmentViewInjector<ExtraData>.self
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/Components.swift:74]: 👏 😅 Happy to see you became friends with punctuation.

**Review by VojtaStavik — APPROVED** [review]
LGTM but the snapshots look a bit off? 🤔 

Screen Shot 2021-05-14 at 10 44 01

**Comment by VojtaStavik** [comment]
@fortmarek or is it just because of the corner masking not working correctly in snapshots? (known issue)

**Comment by fortmarek** [comment]
> @fortmarek or is it just because of the corner masking not working correctly in snapshots? (known issue)

Yes, I have edited that into the PR description

File: Sources/StreamChatUI/Components.swift (hunk 1 above)
Comment (fortmarek) [reply Sources/StreamChatUI/Components.swift:74]: indeed 😄

**Pull request merged**